src/MoBi.Core/OSPSuite.Core/Domain/Builder/ExpressionProfileBuildingBlock.cs

```diff
@@ -49,6 +67,10 @@ public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager clone
       }
    }
 
+   public interface IExpressionProfileBuildingBlock : IBuildingBlock
+   {
+   }
+
    public enum ExpressionType
```

Comment:
I feel that this enum could be made a bit more powerful and have the caption, the icon etc.. so that we don't have to sue all the switches. 
See what we do with Some other enum such as ApplicationType etc. (in PKSim)